extensions/amp-a4a/0.1/refresh-manager.js

```diff
@@ -101,8 +101,9 @@ export class RefreshManager {
   /**
    * @param {!./amp-a4a.AmpA4A} a4a The AmpA4A instance to be refreshed.
    * @param {!RefreshConfig} config
+   * @param {number=} refreshInterval
```

Comment:
Give the units for refreshInterval, e.g. refreshIntervalSecs.

Reply:
Done.